package/ios/Core/CameraSession+Photo.swift

```diff
@@ -63,7 +63,8 @@ extension CameraSession {
       let enableShutterSound = options["enableShutterSound"] as? Bool ?? true
 
       // depth data
-      photoSettings.isDepthDataDeliveryEnabled = photoOutput.isDepthDataDeliveryEnabled
+      photoSettings.isDepthDataDeliveryEnabled = photoOutput.isDepthDataDeliveryEnabled && photoOutput.isDepthDataDeliverySupported
+      photoSettings.embedsDepthDataInPhoto = photoSettings.isDepthDataDeliveryEnabled
```

Comment:
Does that really change something? As per Apple Developer Documentation, the default value is already `true`: 

![image](https://github.com/mrousavy/react-native-vision-camera/assets/15199031/f982a16b-823e-4c9e-8a10-c179f1b118ac)

Did you test this? Does this make a difference?

Reply:
Yes I know it's the default behavior but to know that you have to look it up in the iOS documentation, which is a setback given how rnvc wants to simplify the use of those features. By making it explicit, you can find out why the depth data are being embedded in the photo by simply debugging the code.